sonar-scanner-engine/src/main/java/org/sonar/scanner/postjob/DefaultPostJobContext.java

```diff
@@ -37,21 +37,29 @@
 
 public class DefaultPostJobContext implements PostJobContext {
 
-  private final Settings settings;
+  private final Configuration config;
   private final IssueCache cache;
   private final AnalysisMode analysisMode;
   private InputComponentStore inputComponentStore;
+  private final Settings mutableSettings;
 
-  public DefaultPostJobContext(Settings settings, IssueCache cache, InputComponentStore inputComponentStore, AnalysisMode analysisMode) {
-    this.settings = settings;
+  public DefaultPostJobContext(Configuration config, Settings mutableSettings, IssueCache cache, InputComponentStore inputComponentStore,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)